### DISCUSSION TITLE
error: could not write config file .git/config: Permission denied

### DISCUSSION THREAD

**Discussion by zicohip** [ohmyzsh/ohmyzsh#11397] [Help]
when write `omz update` command in wsl it get 
![image](https://user-images.githubusercontent.com/76248189/208243941-7b80c3fc-e1c0-4f30-a92a-602c0178a9d6.png)

and  
![image](https://user-images.githubusercontent.com/76248189/208243964-801b3708-4159-4b2f-b56c-f4875621c977.png)

**Comment by rwmitchell** [comment]
Have you tried looking to see what is going on?

What are the file permissions for .git/config and who is the owner?
